src/record/key.rs

```diff
@@ -190,6 +190,12 @@ impl RemoteId {
         &self.full_id[..self.provider_len]
     }
 
+    /// Check whether the `provider` part of the remote id is `"local"`.
```

Comment:
Do you feel that the quotes are necessary? Maybe `local` is sufficient.

Reply:
I guess I wanted to make it explicit this is a string literal, unlike `provider`. But I'll rephrase this.